tests/regression/falsifiable/arrays_multidimensional3.lus

```diff
@@ -0,0 +1,11 @@
+node M(const m: int; const n: int) returns (A: [int^m^n, int]);
+let
+  A = {0^2^3, 0};
+tel
+
+node N() returns(A: [int^2^3, int]);
+let
+  A = M(2, 3);
+  check (A = pre A);
```

Comment:
This check is not legal with the latest fix in develop.

Reply:
Addressed